lib/urlwatch/reporters.py

```diff
@@ -978,3 +978,65 @@ def submit(self):
 
         for chunk in chunkstring(text, self.MAX_LENGTH, numbering=True):
             asyncio.run(xmpp.send(chunk))
+
+class ProwlReport(TextReporter):
```

Comment:
The PEP-8 checker complains:

```
/home/travis/build/thp/urlwatch/lib/urlwatch/reporters.py:982:1: E302 expected 2 blank lines, found 1
```

(just add another blank line above the `class ProwlReport...` line.

Maybe also name the class "ProwlReporter" to be in line with other class names.